aarch64_linux/aarch64_ci_setup.sh

```diff
@@ -19,7 +19,16 @@ curl -L -o /mambaforge.sh https://github.com/conda-forge/miniforge/releases/late
 chmod +x /mambaforge.sh
 /mambaforge.sh -b -p /opt/conda
 rm /mambaforge.sh
-/opt/conda/bin/conda config --set ssl_verify False
-/opt/conda/bin/conda install -y -c conda-forge python=${DESIRED_PYTHON} numpy pyyaml setuptools patchelf pygit2 openblas ninja scons
+source /opt/conda/etc/profile.d/conda.sh
+conda config --set ssl_verify False
+conda create -y -c conda-forge -n aarch64_env python=${DESIRED_PYTHON}
```

Comment:
lol this is also an important distinction that an env is now created--not sure if this will have other implications

Reply:
(to be clear, this seems more correct)

Reply:
The current code is inplace modifying the env from being python 3.10 to another python version as asked. For some reason that worked for 3.8/3.9/3.11 but 3.12 changed which packages are in the default distribution and so just doing `/opt/conda/bin/conda install -y -c conda-forge python=${DESIRED_PYTHON}` fails.